test/import_test.js

```diff
@@ -13,7 +17,44 @@ function addTest(name, translatorID, input, expected) {
 
 		assert.equal(response.headers['zotero-translator-id'], translatorID);
 
-		assert.deepEqual(response.body, expected);
+		const items = {
+			expected: flatten(expected.map(item => Zotero.Utilities.itemToAPIJSON(item))), // normalize and flatten results
+			found: response.body,
+		};
+
+		for (const ef of ['expected', 'found']) {
+			for (const item of items[ef]) {
+				// inline attached notes -- notes are exported as separate items with a parentItem, but the parentItem is going to be different every time the test is ran
+				if (!item.notes) {
+					item.notes = items[ef]
+						.filter(note => (note.itemType === 'note' && note.parentItem === item.key))
+						.map(note => ({ note: note.note, itemType: note.itemType}));
+				}
+			}
+			// remove notes now inlined
+			items[ef] = items[ef].filter(item => (item.itemType !== 'note' || !item.parentItem));
+
+			for (const item of items[ef]) {
+				// remove purely administrative fields -- must be done in 2nd run because the key/parentItem are required in the 1st run for inlining notes
+				delete item.key;
+				delete item.version;
+				delete item.parentItem;
+
+				// remove empty arrays to simplify comparison
+				for (const prop of ['tags', 'notes', 'attachments', 'seeAlso']) {
```

Comment:
This is post-`itemToAPIJSON`, so `notes`, `attachments`, and `seeAlso` shouldn't exist.